website/example/Select.demo.js

```diff
@@ -0,0 +1,35 @@
+// import React from 'react';
```

Comment:
Why did you comment this out? Is it because you're missing @igloo-ui/select inside next.config.js?

Reply:
I wasn't able to make this section work, I commented it with a plan of asking for help later and don't have the time to fix + forgot it after.